src/react/account/__tests__/AccountUserList.test.tsx

```diff
@@ -124,5 +140,88 @@ describe('AccountUserList', () => {
     expect(screen.getByText(SAMPLE_USER_NAME)).toBeInTheDocument();
 
     //TODO test copy arn
+    const firstRow = screen.getAllByRole('row')[1];
+
+    const arnButtonOfFirstRow = getAllByRole(firstRow, 'button');
+    expect(getByText(
+      arnButtonOfFirstRow[1],
+      /Copy ARN/i,
+    )).toBeInTheDocument();
+
+    fireEvent.mouseEnter(screen.getAllByRole('button')[4]);
+
+    expect(
+      screen.getByText('Copy ARN'),
+    ).toBeInTheDocument();
+
+    const eyeButtonOfFirstRow = getAllByRole(firstRow, 'button')[0];
+    expect(
+      eyeButtonOfFirstRow.getElementsByClassName('fas fa-eye')[0].className,
+    ).toBeDefined();
+
+    const createdOnOfFirstRow = getAllByRole(firstRow, 'gridcell');
+    expect(getByText(
+      createdOnOfFirstRow[2],
+      /2022-03-02/i,
+    )).toBeInTheDocument();
+
+  });
+  it('should render header buttons and column names', async () => {
+
+    render(<AccountUserList accountName="account" />, {
+      wrapper,
+    });
+
+    fireEvent.click(screen.getAllByText('Loading users...')[0])
+    await waitFor(() => screen.getAllByRole('button')[2])
+
+    /**********           Number of columns :         ************/
+
+    expect(screen.getAllByRole('columnheader').length).toEqual(nbrOfColumnsExpectedWithoutEdit);
+
+    /**********           Buttons 'search' and 'Create user' exist :         ************/
+    const searchZone = screen.getAllByRole('button')[0];
+    expect(
+      searchZone.getElementsByClassName('fas fa-search')[0].className,
+    ).toBeDefined();
```

Comment:
We should better select button by their text as well or accessible labels when we can have several.

Eg. `screen.getByRole('button', {label: 'search'})`